<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteElement</class>
 <widget class="QWidget" name="SpriteElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Details</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sprite Details</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="positionLabel">
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="xEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string notr="true"> X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string notr="true">Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="yEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="wLabel">
       <property name="text">
        <string notr="true">W</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="wEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="hLabel">
       <property name="text">
        <string notr="true">H</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="hEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="borderLabel">
     <property name="text">
      <string>Border</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="3">
      <widget class="QLineEdit" name="borderEditT"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="borderEditL"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="borderT">
       <property name="text">
        <string notr="true">T</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="borderL">
       <property name="text">
        <string notr="true"> L</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="borderR">
       <property name="text">
        <string notr="true">R</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="borderEditR"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="borderB">
       <property name="text">
        <string notr="true">B</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="borderEditB"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pivotLabel">
     <property name="text">
      <string>Pivot</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="pivotX">
       <property name="text">
        <string notr="true">X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pivotXEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="pivotY">
       <property name="text">
        <string notr="true">Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pivotYEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
